treeherder/webapp/api/push.py

```diff
@@ -254,10 +257,17 @@ def health(self, request, project):
         perf_failures = get_perf_failures(push)
         perf_result = 'fail' if len(perf_failures) else 'pass'
 
+        push_result = 'pass'
+        for metric_result in [test_result, lint_result, build_result, perf_result]:
+            if metric_result == 'indeterminate' and push_result != 'fail':
+                push_result = metric_result
+            if metric_result == 'fail':
```

Comment:
Should this second if statement be an OR of the first if statement since it's assigning the same value?

Reply:
Yeah, you're right.  It should be an `elif` instead.  Thanks.  :)